app/presenters/nsm/tasks/case_category.rb

```diff
@@ -0,0 +1,16 @@
+module Nsm
```

Comment:
Are you sure this is what's Mary specced? On the [prototype](https://crmfour-proto-main.apps.live.cloud-platform.service.justice.gov.uk/latest-nsm/provider/form/task-list?question-claimType=complete&question-org=complete&question-client=complete&question-case-details=&question-hearing=&question-case-disposal=&question-just=&question-claim=&question-work=&question-letters=&question-disburse=&question-payment=&question-other=&question-upload=&question-check=&question-equal=&ufnNSM=100323/234&laa-ref=LAA-5a7d3c&claimType=Non-standard%20magistrates%27%20court%20payment&stageProvider=Undesignated%20area&stageCourt=Undesignated%20area&stage=PROG), there is just one "case disposal" task that covers all the screens relating to youth court:
<img width="507" alt="Screenshot 2024-11-27 at 16 44 25" src="https://github.com/user-attachments/assets/66413c64-d146-4c53-9811-485249655df5">

Whereas this seems to be showing a task for each new screen, with a bunch of them marked as "not applicable":
<img width="661" alt="Screenshot 2024-11-27 at 16 46 07" src="https://github.com/user-attachments/assets/d15d585f-e500-472f-8eff-130362667ea1">

Reply:
Agreed with Mary to change in-line with the prototype; but some more thinking to be done so it may change again

Reply:
Obviously don't change anything from this comment but from my perspective on the design, the way it is in the prototype makes more sense than individual items for each form anyways (though this is an opinion): In general, each part of the task list is a collection of forms that relate together (sometimes that's one form, sometimes that's a few) and case disposal, case category and case outcome are closely coupled. The issue feels like the name of the group...I still feel like Case outcome = Case disposal so we should choose one of those (probably Case outcome) and use that for the Case disposal form AND the tasklist group link